Configuration/TSconfig/ContentElementWizard.txt

```diff
@@ -11,3 +11,22 @@ mod.wizards.newContentElement.wizardItems.plugins {
         }
     }
 }
+
+#Marker for the Configuration/TCA/tx_seminars_seminars.php
+TCEFORM.tx_seminars_seminars.categories.PAGE_TSCONFIG_STR = %
```

Comment:
Please let's find a name for that variable that describes more what it does. `PAGE_TSCONFIG_STR` only describes where it is and its type (and also, it's an abbreviation).